views/beers.hbs

```diff
@@ -0,0 +1,3 @@
+{{> nav }}
```

Comment:
The "nav" HTML is used in each page.
Thus, the HTML can be included in the layout instead of in a separate partial.